apps/wpcom-block-editor/src/default/features/rich-text.js

```diff
@@ -65,7 +65,7 @@ const ConnectedRichTextJustifyButton = compose(
 		};
 	} ),
 	withDispatch( ( dispatch ) => ( {
-		updateBlockAttributes: dispatch( 'core/editor' ).updateBlockAttributes,
+		updateBlockAttributes: dispatch( blockEditorStore ).updateBlockAttributes,
```

Comment:
Did you mean to use `editorStore` here? We'll also need the relevant import above.